hawc/apps/study/api.py

```diff
@@ -14,7 +14,9 @@
 from . import models, serializers
 
 
-class Study(viewsets.ReadOnlyModelViewSet):
+class Study(
+    viewsets.ReadOnlyModelViewSet, mixins.CreateModelMixin,
```

Comment:
by convention, mixins come before base models, as they should replace default implementations; order should be switched: https://www.ianlewis.org/en/mixins-and-python

Reply:
Fixed!